apps/include/pcl/apps/nn_classification.h

```diff
@@ -165,8 +167,10 @@ namespace pcl
           if (pcl::io::savePCDFile (file_name.c_str (), *training_features) != 0)
             return (false);
           std::ofstream f (labels_file_name.c_str ());
-          BOOST_FOREACH (int i, labels_idx_)
+          for (int i : labels_idx_)
```

Comment:
`const auto&` would be better

Reply:
I don't use `auto` in case of primitive data types (see `modernize-use-auto` discussion).